test/inductor/test_torchinductor.py

```diff
@@ -2043,9 +2043,20 @@ def fn(a, b):
             b = b.view(-1)
             return torch.cumsum(a, 0) + torch.cumsum(b, 0)
 
-        a = make_tensor(10, 3, 352, 352, low=0, dtype=torch.float32, device=self.device)
-        b = make_tensor(10, 3, 352, 352, low=0, dtype=torch.float64, device=self.device)
-        self.common(fn, (a, b), rtol=1e-4, atol=1e-5, check_lowp=False)
+        dtype_a = torch.float32
+        dtype_b = torch.float64
```

Comment:
Would it be more helpful to set dtype_b to bfloat16 is mps and dtype is supported?

Reply:
I can try making that change, let me see.